buildscript/build.go

```diff
@@ -51,18 +51,12 @@ func TestKerberos(ctx *task.Context) error {
 
 // TestIntegration is an Executor that runs all integration tests for the provided packages.
 func TestIntegration(ctx *task.Context) error {
-	// TODO: Replace func body with following line in TOOLS-2807.
-	// return runTests(ctx, selectedPkgs(ctx), testtype.IntegrationTestType)
-
-	// Don't run integration tests in /common yet.
-	pkgs := selectedPkgs(ctx)
-	for i := range pkgs {
-		if pkgs[i] == "common" {
-			pkgs = append(pkgs[:i], pkgs[i+1:]...)
-			break
-		}
-	}
-	return runTests(ctx, pkgs, testtype.IntegrationTestType)
+	return runTests(ctx, selectedPkgs(ctx), testtype.IntegrationTestType)
+}
+
+// TestSRV is an Executor that runs the SRV connection string test for package `common`.
+func TestSRV(ctx *task.Context) error {
+	return runTests(ctx, []string{"common"}, testtype.SRVConnectionStringTestType)
```

Comment:
```suggestion
	return runTests(ctx, selectedPkgs(ctx), testtype.SRVConnectionStringTestType)
```
I don't think we should hard-code that common is the only package with srv tests. This is the case right now, but a future refactoring could _theoretically_ change this. I think that maintaining consistency with the other test targets will minimize surprise for folks working on this in the future